src/renderer/components/Editor/WordComponent.tsx

```diff
@@ -31,7 +31,7 @@ const makeWordInner = (isDragActive: boolean, isInInactiveTake: boolean) =>
   styled('div')({
     display: 'inline-block',
     cursor: isInInactiveTake ? 'pointer' : 'text',
-    color: colors.white,
+    color: isInInactiveTake ? colors.grey[600] : colors.white,
```

Comment:
is logic enabled all the time (for words that aren't in takes at all) or just for words in takes? (sorry can't tell in the video)

Reply:
`isInInactiveTake` is false if a word is not in a take at all